source/grid/grid_in.cc

```diff
@@ -2760,10 +2787,519 @@ GridIn<dim, spacedim>::parse_format (const std::string &format_name)
 template <int dim, int spacedim>
 std::string GridIn<dim, spacedim>::get_format_names ()
 {
-  return "dbmesh|msh|unv|vtk|ucd|xda|netcdf|tecplot";
+  return "dbmesh|msh|unv|vtk|ucd|abaqus|xda|netcdf|tecplot";
 }
 
 
+namespace internal
+{
+  template <int dim>
+  Abaqus_to_UCD<dim>::Abaqus_to_UCD (void)
```

Comment:
Thanks for the tip: In 52cf17c I moved the entire class definition to the bottom of the `.cc` file and placed it within an anonymous namespace.